packages/web/src/scss/components/TextArea/README.md

```diff
@@ -71,10 +71,29 @@ modifier to fill the available space.
 ## Validation States
 
 Validation states can be presented either by adding a CSS modifier class
-(`TextArea--success`, `TextArea--warning`, `TextArea--error`), or by adding
+(`TextArea--success`, `TextArea--warning`, `TextArea--danger`, `TextArea--error`), or by adding
 a JS interaction class when controlled by JavaScript (`has-success`,
 `has-warning`, `has-error`).
 
+```html
+<div class="TextArea TextArea--danger">
```

Comment:
Again. There should be has-danger class.

Reply:
done